presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -66,8 +72,9 @@ Date and Time Functions
 .. function:: from_iso8601_timestamp(string) -> timestamp with time zone
 
     Parses the ISO 8601 formatted date ``string``, optionally with
-    time and time zone, into a ``timestamp with time zone``. The time defaults
```

Comment:
This seems to be just changing the wrapping. Please remove

Reply:
It adds the milliseconds for correctness as requested by @martint

Reply:
This formatting change makes it hard to see what actually changed. Please leave it alone except for adding the fractional seconds.

Reply:
done